flink/values.yaml

```diff
@@ -288,21 +288,25 @@ zookeeper:
   persistence:
     enabled: true
 
-#secret:
-#  secrets:
+secrets:
+#  Example of plain predefined secrets
+#  kubernetesSecrets:
 #    - name: kerberos
 #      mountPath: /kerberos
-#    - name: kafka-ssl
-#      mountPath: /etc/certs
-#  sealedSecrets:
-#    # The encrypted raw file sealed secrets generated for example with
-#    # kubeseal --raw --from-file=... --controller-name sealed-secrets --scope namespace-wide
-#    sealedSecretFiles:
-#      file1: ...
-#      file2: ...
-#    sealedSecretFilesPath: /etc/sealed
-#    # The encrypted raw env sealed secrets generated for example with
-#    # echo -n password | kubeseal --raw --from-file=/dev/stdin --controller-name sealed-secrets --scope namespace-wide
-#    sealedSecretEnvs:
-#      env1: ...
-#      env2: ...
+  bitnamiSealedSecrets:
+    enabled: false
+    # The encrypted raw file sealed secrets generated for example with
+    # kubeseal --raw --from-file=... --controller-name sealed-secrets --scope namespace-wide
+    sealedSecretFiles:
+      file1:
```

Comment:
This should be commented, like:
```
sealedSecretFiles: {}
# file1: encrypted_file1
```
With current values.yaml, if you pass additional values you can generate non-expected output:

```
>> cat add-values.yaml
secrets:
  bitnamiSealedSecrets:
    enabled: true
    sealedSecretFiles:
      file_test: my_test_file
    sealedSecretEnvs:
      env_test: my_test_env
>> helm template -f  add-values.yaml .
...
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/namespace-wide: "true"
  name: RELEASE-NAME-secretenvs
  namespace: default
spec:
  encryptedData:
    env1:
    env2:
    env_test: my_test_env
---
# Source: flink/templates/sealedsecretfiles.yaml
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/namespace-wide: "true"
  name: RELEASE-NAME-secretfiles
  namespace: default
spec:
  encryptedData:
    file1:
    file2:
    file_test: my_test_file
```